src/Sulu/Bundle/WebsiteBundle/Resolver/StructureResolver.php

```diff
@@ -35,12 +36,24 @@ class StructureResolver implements StructureResolverInterface
      */
     protected $extensionManager;
 
+    /**
+     * @var bool
+     */
+    private $enabledPathAttribute = true;
+
     public function __construct(
         ContentTypeManagerInterface $contentTypeManager,
-        ExtensionManagerInterface $structureManager
+        ExtensionManagerInterface $structureManager,
+        bool $enabledPathAttribute = true
```

Comment:
When reading that code, shouldn't we make that already an array like this:
```php
['path' => true]
```

What if we come up with more attributes we want to get rid of? That would result in quite a massive constructor.